apiary.apib

```diff
@@ -2637,6 +2639,80 @@ enabled. See <a href="/#reference/0/users/create-two-factor-authentication-secre
             "Message": ""
         }
 
+### Login with token [POST /organizations/auth/token]
+
+Login with a user token obtained from `/organizations/auth` endpoint.
```

Comment:
It can be a user token or a device token. Also works with a token obtained from Organizations.RegisterLicense e.g.
Maybe we can change to:

"Login with a Token obtained from another endpoint such as <a href="/#reference/0/organizations/register-license">Register License</a>."

Reply:
@sambhare  updated